Workbooks/Activity Log/ Resource/ActivityLogsInsights.workbook

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
probably not intentional and strange: it LOOKS like this folder has a leading space?  it is not `Resource` but `[space]Resource`?

![image](https://user-images.githubusercontent.com/10158007/159046908-02971665-4a2c-425d-943b-c59cfc47e805.png)

Reply:
thanks. fixed this